metrics/exporter_test.go

```diff
@@ -20,6 +20,7 @@ import (
 )
 
 func TestNewStackdriverExporter(t *testing.T) {
+	resetCurPromSrv()
```

Comment:
What's this change about?

Reply:
That's the workaround for a flaky test. I'm trying to address it in another pr
https://github.com/knative/pkg/pull/138

Reply:
Mhm., waiting for fixing :smirk:, i have removed the work around commit. PTAL